client/app/profilesScreen.tsx

```diff
@@ -1,14 +1,58 @@
-import { View, Text } from 'react-native'
-import React from 'react'
-import { Link } from 'expo-router'
+import { View, Text, StyleSheet, SafeAreaView} from 'react-native'
+import React, { useEffect } from 'react'
+import { Link, router } from 'expo-router'
+import NewProfileButton from '../components/Profiles/NewProfileButton'
+import { useDataContext } from '@/context/globalContext'
+import { getUser } from '@/services/apiService'
+import ListedProfiles from '@/components/Profiles/ListedProfiles'
 
 const profilesScreen = () => {
+  const { setUser  } = useDataContext();
+ 
+  useEffect(() => {
+    //TODO: Change hard-coded userId to be from the logged in user
+    const fetchUser = async () => {
+      const userId = 1; // Replace with dynamic once login is working correctly
+      const userData =  await getUser(userId);
+      setUser( userData);
```

Comment:
I've fetched the user data and profiles in the homepage we can probably get rid of this here